tox.ini

```diff
@@ -31,6 +35,7 @@ deps =
     mock
     slimit
     pillow
+    pywin32
```

Comment:
should this not be prefixed with `mywindows:`?

Reply:
I had issues trying to get that to work in my testing. got various results including it saying something like invalid package pywin3 or :pywin. feel free to try but I couldn't get it right and with ONLY adding mywindows: pywin32 it didn't work if I remeber right.

Reply:
Well we cannot have this in tox.ini if it breaks on linux. 

Could it be related to this issue: https://github.com/mhammond/pywin32/issues/1177

Reply:
not sure how thats related thats a issues, as stated in that issue, with pywin32's setup. I havn't even gotten it to go to that step. it errors trying to install a module not even in the list(pywin3, NOT pywin32) hence why its not valid and fails.

Reply:
If you could provide the actual error that would be helpful.

Reply:
I think there is a bug in tox when reusing deps, I was testing with `tox -e flake8` and this workaround worked:

```diff
--- tox.ini
+++ tox.ini
@@ -17,9 +17,9 @@ minversion = 2.0
 
 [testenv]
- platform =
-           mylinux: linux
-           mymacos: darwin
-           mywindows: win32
 install_command = pip install --ignore-installed {opts} {packages}
 passenv = DISPLAY PYTHONPATH
 setenv = PYTHONPATH = {toxinidir}
@@ -35,7 +35,6 @@ deps =
     mock
     slimit
     pillow
-    mywindows: pywin32
 whitelist_externals = pytest bash
 commands = {envpython} setup.py test
 
@@ -78,8 +77,13 @@ commands =
     pytest -v --basetemp=_pytest_temp -s deluge/plugins
 
 [testenv:py27]
+platform =
+    mylinux: linux
+    mymacos: darwin
+    mywindows: win32
 deps =
     {[testenv]deps}
+    mywindows: pywin32
     py2-ipaddress
 basepython = python2.7
 commands = {[testenv:pydef]commands}

```

All other combinations failed. So you need to specify pywin32 in each env

Reply:
unless I did it wrong seems to work, besides a minor mistake I made, on travis(linux) but still a no go on appveyor(windows).